templates/stevenage/foot.html

```diff
@@ -15,33 +15,46 @@
 	<div class="secondaryFooterWrapper">
 		<div id="secondaryFooter" class="group">
 			   		
-			<form action="http://www.govmetric.com/WebMetric/websatisfaction.aspx" method="post">
-				<div class="ratePage">
-					<input type="hidden" name="organisationid" value="343" />
-					<input type="image" src="http://www.govmetric.com/images/green-face-30x30.gif" alt="Rate as Good" name="metric1" value="1" />
-					<input type="image" src="http://www.govmetric.com/images/amber-face-30x30.gif" alt="Rate as Average" name="metric2" value="2" />
-					<input type="image" src="http://www.govmetric.com/images/red-face-30x30.gif" alt="Rate as Poor" name="metric3" value="3" />
-					<p class="inst"><img src="http://www.stevenage.gov.uk/Images/rateThisPage.gif" alt="Please rate this page." /></p>
-				</div>
+<!-- WebMetric snippet starts here -->
+
+<div class="gm_sidebar">
+    <a
```

Comment:
For some reason these images don't appear on Safari for me (http://davea.petitions.dev.mysociety.org):
<img width="197" alt="screen shot 2017-08-02 at 16 03 53" src="https://user-images.githubusercontent.com/4776/28880203-38c80336-779c-11e7-99e4-712587196adf.png">

However, they're fine on Chrome. Not sure what's going on there.
<img width="262" alt="screen shot 2017-08-02 at 16 03 47" src="https://user-images.githubusercontent.com/4776/28880199-36dc9302-779c-11e7-9471-0df7c7f80b95.png">